pkg/tokens/plugin.go

```diff
@@ -108,4 +109,20 @@ type TokenPool struct {
 
 	// Standard is the well-defined token standard that this pool conforms to (optional)
 	Standard string
+
+	// Event contains info on the underlying blockchain event for this pool creation
+	Event blockchain.Event
+}
+
+type TokenTransfer struct {
+	fftypes.TokenTransfer
+
+	// PoolProtocolID is the ID assigned to the token pool by the connector
+	PoolProtocolID string
+
+	// Event contains info on the underlying blockchain event for this transfer
+	Event blockchain.Event
```

Comment:
It feels slightly odd to me having the tokens plugin refer to the blockchain plugin (rather than exclusively `fftypes`).

However, I understand if `blockchain.Event` moved to `fftypes` it would need to be distinguished from `fftypes.BlockchainEvent`.

Reply:
Let's go with this for now - journey makes sense